0000-named-tracers.md

```diff
@@ -0,0 +1,67 @@
+# Named Tracers 
+
+_Creating tracers using a factory mechanism and naming those tracers in accordance with the (logical) components being traced._
+
+## Suggested reading
+
+* [Proposal: Tracer Components](https://github.com/open-telemetry/opentelemetry-specification/issues/10)
+* [Global Instance discussions](https://github.com/open-telemetry/opentelemetry-specification/labels/global%20instance)
+
+## Motivation
+
+Instead of having a global tracer singleton, every library needs to create its own tracer, which also has a proper name that identifies the library / logical component that uses this tracer.
```

Comment:
"every library" -> "every instrumentation library"

Reply:
"identifies the library / logical component" -> "instrumentation library".

I would add. If there are two different instrumentation-libraries for the same technology (e.g. MongoDb), these instrumentation-libraries should have distinct names.

Reply:
Addressed in follow-up commit.